packages/flutter/lib/src/material/app.dart

```diff
@@ -947,6 +948,24 @@ class _MaterialAppState extends State<MaterialApp> {
     );
   }
 
+  Widget _moveExitWidgetSelectionButtonBuilder(
+    BuildContext context, {
+    required VoidCallback onPressed,
+    bool isLeftAligned = true,
+  }) {
+    return IconButton(
+        padding: const EdgeInsets.all(-8.0),
+        iconSize: 32,
```

Comment:
nit: can we put 32 and 40 into named consts?

Reply:
Yes, done!